### PR TITLE
DFTDP # 203 - Upgrade shared library

### PR DESCRIPTION
**Pull request by shruthi-rapala**
Bump ejs from 3.1.9 to 3.1.10 in /shared-portal-ui

# Description

This PR includes the following proposed change(s):

- {List all the changes, if possible add the jira ticket #}

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
**- [ ] Version change**

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

### Screenshots

Include screen captures or recordings with captions of the tests run (unit tests, postman request/response and or manual UI driven tests.

## Does the change impact or break the Docker build?

- [ ] Yes
- [ ] No

If Yes: Has Docker been updated accordingly?

- [ ] Yes
- [ ] No

## Checklist:

- [ ] My code follows the [style guidelines](https://justice.gov.bc.ca/wiki/display/JAM/RCC+Style+Guide) of this project
- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: driver-portal/src/ClientApp/src/app/submission-requirements/submission-requirements.component.html [hunk 1]
```diff
@@ -45,128 +45,139 @@
...
-                documentType.name
-              }}</mat-option>
+          <div style="padding-top: 24px">
+            <mat-form-field appearance="outline" style="width: 350px">
+              <mat-label>Select Document Type</mat-label>
+              <mat-select
+                name="documentType"
+                [formControl]="uploadForm.controls.documentSubType"
+              >
+                @for (documentType of documentSubTypes; track documentType) {
```
Comment (jburditt) [inline driver-portal/src/ClientApp/src/app/submission-requirements/submission-requirements.component.html:55]: not necessary but you could clean this up by using ngFor https://angular.io/api/common/NgFor#description

File: driver-portal/src/ClientApp/src/app/submission-requirements/submission-requirements.component.html [hunk 2]
```diff
@@ -45,128 +45,139 @@
...
+                <mat-option [value]="documentType.id">{{
+                  documentType.name
+                }}</mat-option>
+                }
+              </mat-select>
+              @if (uploadForm.controls.documentSubType.getError('required')) {
```
Comment (jburditt) [inline driver-portal/src/ClientApp/src/app/submission-requirements/submission-requirements.component.html:61]: same here, could use ngIf

**Review by jburditt — APPROVED** [review]

File: driver-portal/src/ClientApp/src/app/submission-requirements/submission-requirements.component.html (hunk 1 above)
Comment (shruthi-rapala) [reply driver-portal/src/ClientApp/src/app/submission-requirements/submission-requirements.component.html:55]: I have been using the new control flows in angular where Ngif, Ngfor are replaced with @if, @for  conditional expression from angular 16 https://angular.io/guide/control_flow

File: driver-portal/src/ClientApp/src/app/submission-requirements/submission-requirements.component.html (hunk 2 above)
Comment (shruthi-rapala) [reply driver-portal/src/ClientApp/src/app/submission-requirements/submission-requirements.component.html:61]: I have been using the new control flows in angular where Ngif, Ngfor are replaced with @if, @for  conditional expression from angular 16 https://angular.io/guide/control_flow

**Pull request merged**